Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1-alpine AS build-env
+
+RUN apk add --no-cache --upgrade git openssh-client ca-certificates
```

Comment:
Hi @marcacohen, this is required for the `go get -u github.com/google/addlicense` command to succeed (since this is done over HTTPS).
I can recheck regarding the openssh-client, but I'm almost sure that it didn't work without it.

You can see other examples of the same thing on other repositories, e.g.:
https://github.com/projectdiscovery/httpx/blob/master/Dockerfile

Thanks!

Reply:
Ok, that makes sense. Thanks for the PR and sorry for the delay!